### PR TITLE
Support 1058

### PR DESCRIPTION
**Pull request by raynaudoe**
Added new types for runtime 1058 support.

### CODE REVIEW

File: app/Makefile.version [hunk 1]
```diff
@@ -1,3 +1,3 @@
 APPVERSION_M=0
-APPVERSION_N=1042
+APPVERSION_N=1058
 APPVERSION_P=2
```
Comment (jleni) [inline app/Makefile.version:3]:
```suggestion
APPVERSION_P=0
```

File: app/src/polkadot_types.c [hunk 2]
```diff
@@ -65,19 +73,55 @@ parser_error_t _readCompactSubId(parser_context_t *c, pd_CompactSubId_t *v) {
...
+    return parser_ok;
+}
+
+parser_error_t _readAttestedCandidate(parser_context_t *c, pd_AttestedCandidate_t *v) {   
     return parser_not_supported;
 }
 
-parser_error_t _readBalanceOf(parser_context_t *c, pd_BalanceOf_t *v) {
+parser_error_t _readBalanceOf(parser_context_t *c, pd_BalanceOf_t *v) {   
     GEN_DEF_READARRAY(16)
```
Comment (jleni) [inline app/src/polkadot_types.c:121]: I am a bit unsure about this one here

File: app/src/polkadot_types.c [hunk 3]
```diff
@@ -481,7 +677,7 @@ parser_error_t _toStringBalanceOf(
         char *outValue,
         uint16_t outValueLen,
         uint8_t pageIdx,
-        uint8_t *pageCount) {
+        uint8_t *pageCount) {   
     MEMSET(outValue, 0, outValueLen);
     MEMSET(bufferUI, 0, sizeof(bufferUI));
```
Comment (jleni) [inline app/src/polkadot_types.c:682]: Ok, here the toString is clear. I forgot about this.
Maybe it is a good idea to create a #define BALANCEOF_LEN 16 or similar to link the _read and the 16 in `bignumLittleEndian_to_bcd(bcdOut, bcdOutLen, v->_ptr, 16);`

File: app/src/polkadot_types.c [hunk 4]
```diff
@@ -65,19 +73,55 @@ parser_error_t _readCompactSubId(parser_context_t *c, pd_CompactSubId_t *v) {
     return _readCompactInt(c, v);
 }
 
-parser_error_t _readAccountId(parser_context_t *c, pd_AccountId_t *v) {
+parser_error_t _readAccountId(parser_context_t *c, pd_AccountId_t *v) {
```
Comment (jleni) [inline app/src/polkadot_types.c:76]: Let's remove all these ending spaces so the diffs are smaller

Comment (raynaudoe) [reply app/src/polkadot_types.c]: Ok, those should be fixed now

**Pull request merged**